packages/charts/src/components/TimelineChart/TimelineChart.tsx

```diff
@@ -1,7 +1,7 @@
 import type { CommonProps } from '@ui5/webcomponents-react';
 import { throttle } from '@ui5/webcomponents-react-base';
-import type { CSSProperties, ReactNode } from 'react';
-import React, { forwardRef, useEffect, useRef, useState } from 'react';
+import { forwardRef, useEffect, useRef, useState } from 'react';
+import type * as React from 'react';
```

Comment:
I would prefer to stay in our common pattern here: `import type { CSSProperties, ReactNode } from 'react';`

Reply:
This is not possible here, as the component imports `MouthEvent` both from typescript and React. 

```ts
React.MouseEvent<HTMLDivElement, MouseEvent>
```

Using the named import of `CSSProperties and `ReactNode` and keeping the namespace import for `MouseEvent` also doesn't work as we would then have multiple `type` imports of the same package.